### PR TITLE
PR for #3: Modify code to run an additional regression only on a subset of years

### LINKED ISSUE #3: Modify code to run an additional regression only on a subset of years
This issue is part of item 5 in [Practice Task 1: Basic functionality](https://github.com/gentzkow/lab-manual-archive/wiki/Practice-Task). Per the task description, steps are as follows:

- [x] Modify the code in `analysis/code/analyze_data.py` to run an additional regression only on the subset of years greater or equal to 1960
- [x] Export the model coefficient, standard errors and p-value into a new table output
- [x] Add new table to the paper

### PR DESCRIPTION
**Pull request by lucamlouzada**
**Closes:** Issue https://github.com/lucamlouzada/template/issues/3. Merges issue branch with main.

**Changes in this branch:**
- Code in `analysis/code/analyze_data.py` was changed to run an additional regression only on years greater than or equal to 1960
- Input txt file in `paper_slides` was updated to include the new table
-  `paper_slides/make.py` was updated to add  `tablefill` call for new table
-  Tex file for new table was added to `paper_slides/code`

**Deliverables:**
- [New table](https://github.com/lucamlouzada/template/blob/eebcd95faa97936d3a60a22d9b574b87b6652879/paper_slides/output/tables_subset_filled.tex)
- [Updated paper](https://github.com/lucamlouzada/template/blob/eebcd95faa97936d3a60a22d9b574b87b6652879/paper_slides/output/paper.pdf)
- [Draftable difference](https://draftable.com/compare/sSubFeoqfRtH)

@ShiqiYang2022 , please review! Thanks :)

### CODE REVIEW

File: paper_slides/code/slides.log
Comment (ShiqiYang2022) [inline paper_slides/code/slides.log:1]: @lucamlouzada Please check whether this is stray file, and if so could you please remove them? Thanks!

(Also for all other relevant files.)

**Review by ShiqiYang2022 — CHANGES_REQUESTED** [review]
Thanks @lucamlouzada for the great work here! 

I left my comments in the conversations. Two additional comments:

* It would be great to revise your https://github.com/lucamlouzada/template/pull/4#issue-2511183056 to display files' permanent links (ref [here](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files)) on the branch.

* We always need to keep branches up to date to `master` before we implement the merge. 
To do this, could you please: (1) Merge `master` into this branch `3-modify-code-to-run-an-additional-regression-only-on-a-subset-of-years`, and [resolve any conflicts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github). (2) Re-run the repo from beginning to end, and post a draftable diff against new `master` branch. (3) Tag @ShiqiYang2022 to review the changes again.


Thanks!

File: paper_slides/code/slides.log
Comment (lucamlouzada) [reply paper_slides/code/slides.log:1]: Yes, it was. Thanks! Deleted in [de0935a](https://github.com/lucamlouzada/template/pull/4/commits/de0935a90dd0309532a97ef7100fbb5a3a89cd65)

**Comment by lucamlouzada** [comment]
Hi @ShiqiYang2022, I have re-run the whole repo in [48a8271](https://github.com/lucamlouzada/template/pull/4/commits/48a82716778cde53a666c9130faa08a2af1ab475) to account for changes in the different branches. First I fixed the conflicts arrising from the fact that #2 led to changes in the main branch. For that, I kept the current branch files in [de0935a](https://github.com/lucamlouzada/template/pull/4/commits/de0935a90dd0309532a97ef7100fbb5a3a89cd65). Then, I re-ran the whole repo which generated new output files combining the changes in #1 and #3 in [48a8271](https://github.com/lucamlouzada/template/pull/4/commits/48a82716778cde53a666c9130faa08a2af1ab475). I also edited the original comment in #4 to use permalinks as suggested. Can you please review? Thanks!

**Review by ShiqiYang2022 — APPROVED** [review]
@lucamlouzada Thanks! All the changes looks great to me so I am approving!

Feel free to merge back to master, then post a summary comment with the stable link for the issue branch. Thanks!

**Comment by lucamlouzada** [comment]
## Summary and deliverables:
In this issue (#3)  I changed the code in `analysis/code/analyze_data.py` to run an additional regression only on years greater than or equal to 1960. As discussed in #4, there were some conflicts due to changes committed in #2, which were first solved and then the whole repo was re-run to combine the updated files.

**Deliverables:**
- [New table](https://github.com/lucamlouzada/template/blob/master/paper_slides/code/tables_subset.tex)
- [Updated paper](https://github.com/lucamlouzada/template/blob/master/paper_slides/output/paper.pdf)

These changes were merged to master in [48a8271](https://github.com/lucamlouzada/template/pull/4/commits/48a82716778cde53a666c9130faa08a2af1ab475) with the corresponding [issue branch](https://github.com/lucamlouzada/template/tree/48a82716778cde53a666c9130faa08a2af1ab475/).

**Comment by lucamlouzada** [comment]
@ShiqiYang2022, please let me know whether the summary is OK so I can close the issue. Thanks again for all your help!

**Comment by ShiqiYang2022** [comment]
@lucamlouzada The summary is great! Cleared to close.

**Pull request merged**